components/Header.tsx

```diff
@@ -215,6 +215,11 @@ export default function Header() {
           {notificationOpen === true ? (
             <div className="notification-menu">
               <ul>
+                <Link href="https://www.eventbrite.it/e/open-source-day-2023-tickets-441134303577">
```

Comment:
<Link target="_blank" href="https://www.eventbrite.it/e/open-source-day-2023-tickets-441134303577">

Reply:
Il link a siti esterni deve aprirsi in un'altra pagina